### PR TITLE
Add delivery for lab manager

### PR DESCRIPTION
**Pull request by morfi11**

### CODE REVIEW

File: src/app/families/families.ts [hunk 1]
```diff
@@ -245,8 +247,20 @@ export class Families extends IdEntity {
       fd.deliveryComments.value = settings.comment;
       fd.distributionCenter.value = settings.distCenter;
       fd.courier.value = settings.courier;
+      if (settings.deliverStatus) fd.deliverStatus.value = settings.deliverStatus;
+      if (settings.archive) fd.archive.value = settings.archive;
       if (settings.selfPickup)
         fd.deliverStatus.value = DeliveryStatus.SelfPickup;
+
+      '\nsettings.basketType: ' + settings.basketType,
```
Comment (noam-honig) [inline src/app/families/families.ts:255]: ?????

**Review by noam-honig — CHANGES_REQUESTED** [review]

File: src/app/families/families.ts (hunk 1 above)
Comment (morfi11) [reply src/app/families/families.ts:255]: Some logging for this interface mostly for lab manager integration. We can remove later when the systems are stable.

Comment (morfi11) [reply src/app/families/families.ts]: very strange. this code worked locally and on mlt-test before I created the PR. 
fixed now and verifying.

**Comment by noam-honig** [comment]
Remove the logging.
I have thousands of theses each day

**Comment by morfi11** [comment]
yalla ten barosh

**Pull request merged**